pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,48 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infoBox = container.find('.info');
+    var content = infoBox.html();
```

Comment:
You could also do `var content = container.find('.info').html();` and avoid defining a variable.